trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name[0..10]
+  end
```

Comment:
- [OPTIONAL] The requirement states that the `correct_name` method of the `TrimmerDecorator` class should return a maximum of ten characters of the name variable of the `nameable`. The logic you wrote returns eleven characters instead of ten. To correct this, you need to set your range between `[0..9]` instead of `[0..10]`. 👍

Reply:
@Gambit142 thank you very much for your suggestion!